### PR TITLE
[webgpu] Add tests to observe the effect of input shape on performance

### PR DESCRIPTION
**Pull request by haoyunfeix**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6301)

### CODE REVIEW

**Comment by haoyunfeix** [comment]
@qjia7 @xhcao @axinging @gyagp PTAL

File: tfjs-backend-webgpu/perf/tune.js [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+  result.uid = `${STATE.name}_${STATE.input}_${STATE.iteration}`
+  result.name = STATE.name;
+  result.backend = STATE.backend;
+  result.input = STATE.input;
+  result.iteration = STATE.iteration;
+  result.webgpu = `${parseFloat(kernel_webgpu.kernelTimeMs).toFixed(2)}`; // WebGPU
+  result.webgl = `${parseFloat(kernel_webgl.kernelTimeMs).toFixed(2)}`; // WebGL
+  result.webglComp = `${parseFloat((result.webgl / result.webgpu) * 100).toFixed(0)}`; // WebGLComp
+  result.scale = `${dimAOuter * dimInner * dimBOuter}`;  // scale
+  result.program = `${kernel_webgpu.extraInfo.split(':')[0]}`;  // scale
```
Comment (qjia7) [inline tfjs-backend-webgpu/perf/tune.js:210]: nit: `// scale` wrong annotation for `result.program`?

File: tfjs-backend-webgpu/perf/tune.js [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+  input: '144,256,256',
+  iteration: 500,
+};
+
+const resultMap = new Map();
+let webglCompResults = [];
+
+let SCALES = [1016064, 5013504, 10838016, 33554432];
+let currentScale = [1016064];
+let customInputs = ['720, 512, 1536', '1, 704, 2000', '1, 128, 1404', '1, 1280, 1001'];
```
Comment (qjia7) [inline tfjs-backend-webgpu/perf/tune.js:370]: It seems that the customInputs and the other test set are run together. There is no place to change the customInputs or only run the customInputs.

Comment (qjia7) [inline tfjs-backend-webgpu/perf/tune.js:370]: It seems that the customInputs and the other test set are run together. There is no place to change the customInputs or only run the customInputs.

File: tfjs-backend-webgpu/perf/tune.js (hunk 1 above)
Comment (haoyunfeix) [reply tfjs-backend-webgpu/perf/tune.js]: Thanks, done.

File: tfjs-backend-webgpu/perf/tune.js (hunk 2 above)
Comment (haoyunfeix) [reply tfjs-backend-webgpu/perf/tune.js:370]: Good point, will add as a feature in separate PR.

**Review by qjia7 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**